shadowfinder/shadowfinder.py

```diff
@@ -138,28 +164,44 @@ def find_shadows(self):
 
         valid_sun_altitudes = pos_obj["altitude"]  # in radians
 
-        # Calculate the shadow length
-        shadow_lengths = self.object_height / np.apply_along_axis(
-            np.tan, 0, valid_sun_altitudes
-        )
+        if self.sun_altitude_angle is not None:
+            # Show relative difference between sun altitudes
+            sun_relative_altitude_angle_difference = (
```

Comment:
`sun_relative_altitude_angle_difference` can become `location_likelihoods` then you avoid the assignment at the end of the block.

Same for `shadow_relative_length_difference` in the next block.